bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/i18n/unifi_nl.properties

```diff
@@ -82,7 +82,7 @@ channel-type.unifi.poeMode.label = PoE-modus
 channel-type.unifi.poeMode.description = De PoE-modus waarin de poort zich bevindt.
 channel-type.unifi.poeMode.state.option.off = Uit
 channel-type.unifi.poeMode.state.option.auto = Automatisch
-channel-type.unifi.poeMode.state.option.24v = 24V
```

Comment:
This file should be omitted, and strings should be updated in Crowdin after PR has been merged.